packages/base-styles/_mixins.scss

```diff
@@ -202,6 +202,11 @@
 	}
 }
 
+@mixin dashed-placeholder-style() {
```

Comment:
Might we consider `placeholder-style` as the name instead, just incase we decide to change the border style in the future? 🤔